Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -59,7 +59,7 @@
     <% checksums.each do |checksum_type, checksum_values| %>
     <% checksum_values.each do |checksum_value| %>
     <% checksum, macos = checksum_value.shift %>
-    <%= checksum_type %> "<%= checksum %>" => :<%= macos %><%= "_or_later" if Homebrew.args.or_later? %>
+    <%= checksum_type %> "<%= checksum %>" => :<%= macos %><%= "_or_later" if Homebrew.global_args.or_later? %>
```

Comment:
What does the above code do?

Reply:
Outputs the `brew bottle` output and appends `or_later` if that argument is specified.

Reply:
Pretty sure this shouldn't be `global_args`?